pkg/patterns/declarative/reconciler.go

```diff
@@ -134,6 +136,40 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
+	// dynamic config
+	dynamicClientset, err := dynamic.NewForConfig(r.config)
+	if err != nil {
+		log.Error(err,"Unable to create dynamic client")
+		return reconcile.Result{}, err
+	}
+
+	newItems := []*manifest.Object{}
+	for _, obj := range objects.Items {
+
+		// Uses unsafe method?? Is it safe?
```

Comment:
I think you can use RESTMapping, like here https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/master/pkg/patterns/declarative/pkg/watch/dynamic.go#L64